releasenotes/notes/remove-tracer-deprecations-c0abaecda9c24b79.yaml

```diff
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - |
+    Remove deprecated attribute ``ddtrace.Tracer.debug_logging``. Use ``logging.setLevel`` instead.
```

Comment:
Not sure `logging.setLevel` is clear enough of an action item?

```python
ddtrace_logger = logging.getLogger("ddtrace")
ddtrace_logger.setLevel(logging.DEBUG)
```

(☝🏻 needs to be confirmed works if we decide to include this action item/steps)